src/Uno.UWP/System/Profile/AnalyticsVersionInfo.iOS.cs

```diff
@@ -0,0 +1,19 @@
+#if __IOS__
+using UIKit;
+
+namespace Windows.System.Profile
+{
+	public partial class AnalyticsVersionInfo
+	{
+		private const string OsName = "Apple";
```

Comment:
You've changed this in the doc file, but not here

Reply:
https://media1.tenor.com/images/9e71e6732539531d7b427c4d40224170/tenor.gif?itemid=4513776

Reply:
http://giphygifs.s3.amazonaws.com/media/jUwpNzg9IcyrK/giphy.gif